frontend/src/components/doctor/PatientSelection.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import { NumberInput, Button, addToast } from "@heroui/react";
+import { fetchUserById } from "../../api/userFunctions";
+import PatientInfo from "./PatientInfo";
+import PastPrescriptions from "./PastPrescriptions";
+import PastMedicalRecords from "./PastMedicalRecords";
+
+export default function NewPrescriptionMain() {
```

Comment:
Rename to something else? For example, following the file name, PatientSelection